bases/renku_data_services/data_api/main.py

```diff
@@ -71,8 +71,8 @@ def create_app() -> Sanic:
         logger.info("enabling sentry")
 
         def filter_error(
-            event: sentry_sdk._types.Event, hint: sentry_sdk._types.Hint
-        ) -> sentry_sdk._types.Event | None:
+            event: "sentry_sdk._types.Event", hint: "sentry_sdk._types.Hint"
+        ) -> "sentry_sdk._types.Event | None":
```

Comment:
No need to quote `None`:

```suggestion
        ) -> "sentry_sdk._types.Event" | None:
```

Reply:
that is not valid code. 
[The | operand cannot be used at runtime to define unions where one or more members is a forward reference. For example, int | "Foo", where "Foo" is a reference to a class not yet defined, will fail at runtime. For unions which include forward references, present the whole expression as a string, e.g. "int | Foo". ](https://docs.python.org/3/library/stdtypes.html#union-type)